server/repository/src/migrations/v2_04_00/item_variant.rs

```diff
@@ -0,0 +1,57 @@
+use crate::migrations::types::{DATETIME, DOUBLE};
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "item_variant"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        // Item variant Table
+        sql!(
+            connection,
+            r#"
+            CREATE TABLE item_variant (
+                id TEXT PRIMARY KEY NOT NULL,
+                name TEXT NOT NULL,
+                item_link_id TEXT NOT NULL REFERENCES item_link(id),
+                --TODO temperature_range_id TEXT REFERENCES temperature_range(id),
```

Comment:
This actually raises a little question for me - is your thinking to add the column here and reset DB seeing as we're in dev, vs an `alter table add column` in another fragment? Only do that once we're in RC? I had been just always creating migrations, but it's an interesting balance (dev experience during the release, vs reading back through migrations and seeing the related columns all together 👀)

Reply:
I think he's just waiting for me, in which case we'd probably update this PR before merge? Anyway, I'll publish the Repository layer of mine shortly.

Reply:
Yup, rebased now so should be all good once both are merged!

Feel free to merge without me if you want this in Develop!